pkg/jobrunaggregator/jobrunaggregatorcachebuilder/cmd.go

```diff
@@ -0,0 +1,99 @@
+package jobrunaggregatorcachebuilder
```

Comment:
Another suggestion here to axe this with a separate commit for history, but get it out of the code for now.